pkg/gofr/gofr_test.go

```diff
@@ -361,6 +361,19 @@ func Test_initTracer(t *testing.T) {
 		"TRACER_URL":     "https://tracer-service.dev",
 	})
 
+	mockConfig8 := config.NewMockConfig(map[string]string{
```

Comment:
Instead of adding more configs like these, we can have a helper function:

```go
createMockConfig := func(traceExporter, host, port, url, authKey string) config.Config {
        return config.NewMockConfig(map[string]string{
            "TRACE_EXPORTER": traceExporter,
            "TRACER_HOST":    host,
            "TRACER_PORT":    port,
            "TRACER_URL":     url,
            "TRACER_AUTH_KEY": authKey,
        })
    } 
```    
    
and then call it like this: **createMockConfig("otlp", "localhost", "2005", "", "valid-token")**

This avoids unnecessary code duplication